rhobs/alerting/data_plane/prometheus.latency_image_repository_provision_alerts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-latency-image-repository-provision-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: latency-image-repository-provision
+    interval: 1m
+    rules:
+    - alert: LatencyImageRepositoryProvision
+      expr: |
+        (
+          (
+            increase(redhat_appstudio_imagecontroller_image_repository_provision_time_bucket{le="+Inf"}[5m])
+            - ignoring(le)
+            increase(redhat_appstudio_imagecontroller_image_repository_provision_time_bucket{le="30"}[5m])
+          ) / ignoring(le)
+          increase(redhat_appstudio_imagecontroller_image_repository_provision_time_bucket{le="+Inf"}[5m])
+        ) > 0.05
+      for: 1m
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
```

Comment:
I suggest including the calculated value (`{{ $value | humanizePercentage }}`) in the summary/description.

Reply:
we don't need to report specific percentage because if alert goes above threshold it is already issue for us

Reply:
It can be helpful also for dev purposes -- will help making sure that the calculations you do are per your expectations.

Anyhow, up to you.